.eslintrc.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
     'react/no-find-dom-node': 'off',
 
     'jsx-a11y/no-autofocus': 'off', // We are a library, people do what they want.
+    'prefer-arrow-callback': ['error', { allowNamedFunctions: true }],
```

Comment:
This is also required for `react-dev-tools` to generate a meaningful display name for `forwardRef` components. The correct exception would be to allow them only in `forwardRef`.